### PR TITLE
[skip ci] Remove WinSCP from package

### LINKED ISSUE #2204: remove winscp from shipping bundle
The winscp binary that is pulled into the bundle uses OpenSSL 1.01 which has published CVEs.
While this is only used for a single point-in-time setup it's still not acceptable and nor is download of the binary without checking thumbprints.

There is a wider question about whether this should be in the main bundle at all. While we need to have a lowest common denominator solution as a fallback for deploying the plugins I am of the opinion that:
a. it doesn't need to be shipped in the main bundle
b. we should not be shipping winscp regardless

### PR DESCRIPTION
**Pull request by jooskim**
This PR fixes #2204 in that it removes the WinSCP binary and updates the affected scripts for vCenter Windows. This will require @stuclem to update the current documentation on UI installation via PR #2261.

### CODE REVIEW

File: ui/installer/vCenterForWindows/install.bat [hunk 1]
```diff
@@ -92,4 +79,12 @@ FOR /F "tokens=*" %%A IN (..\vCenterForWindows\_scratch_flags.txt) DO (
 cd ..\vCenterForWindows
 DEL _scratch_flags.txt
 
-ECHO VIC UI was successfully installed. Make sure to log out of vSphere Web Client if you are logged in, and log back in.
+IF /I %vic_ui_host_url% EQU NOURL (
+    ECHO =============================
+    ECHO With the current version of VIC, you have to manually copy the com.vmware.vicui.* folder in \ui\vsphere-client-serenity to %VMWARE_CFG_DIR%\vsphere-client\vc-packages\vsphere-client-serenity
+    ECHO Note: If you are running vCenter Server 5.5, copy the folder to %PROGRAMDATA%\VMware\vSphere Web Client\vc-packages\vsphere-client-serenity instead
```
Comment (andrewtchin) [inline ui/installer/vCenterForWindows/install.bat]: can the script detect the version and display the correct message?

**Comment by andrewtchin** [comment]
lgtm

File: ui/installer/vCenterForWindows/install.bat (hunk 1 above)
Comment (jooskim) [inline ui/installer/vCenterForWindows/install.bat]: yeah. good suggestion. this is already part of the ui build process where it gets the version from `vic-machine`

Comment (jooskim) [inline ui/installer/vCenterForWindows/install.bat]: Never mind. I mistook the VIC version for VC version. Not sure how to get vCenter version. Will research if there's a way. It is still possible by running `vic-machine inspect` and extracting field `VMware OS version`, but that would require the user to put in the VCH's endpoint information which could be cumbersome

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

**Comment by jooskim** [comment]
♻️ 
Thanks @andrewtchin @caglar10ur

**Comment by jooskim** [comment]
@stuclem The change is now merged, so you can update the documentation.

**Comment by stuclem** [comment]
@jooskim will do. Thanks for the heads-up.

**Pull request merged**